### ISSUE TITLE
Making blacklist for susScr11 with few input bams

### ISSUE THREAD

**Issue by AnnaAMonaco** [Boyle-Lab/Blacklist#36]
Hi, I am woking on a evodevo project and I need to compare mouse and pig ATAC-seq data. For the basic qc of the ATAC I wanted to remove blacklisted regions from my species; I know there is a blacklist for mouse mm10, but not one for pig susScr11. 

To keep the analysis comparable and best-practice-compliant, I would like to generate a blacklist for susScr11, but I only have 12 input bam files, and I read in other discussions here that you recommend >100. So my first question would be if in your opinion it is worth generating this blacklist to use in qc, or if it would actually introduce more bias/artefacts? Would it then be better to just skip the blacklisting for both species before analysis?

If you do think it's worth generating the blacklist for susScr11, my second question is regarding what needs to be in the `mappability/` directory. I have never worked with these umap files, are they the ones that can be generated using [this](https://github.com/hoffmangroup/umap) tool?

Thank you in advance!
Anna

**Comment by aboyle** [comment]
I don't think that you will get a functional blacklist with this tool from only 12 input bams, so it is not worth generating that. As far as bias introduced, either way you are going to have some artifacts from regions that should have been blacklisted.
